Ginger/GingerCoreNET/Run/Context.cs

```diff
@@ -8,17 +9,90 @@
 
 namespace Amdocs.Ginger.Common
 {
-    public class Context
+    public class Context : INotifyPropertyChanged
     {
-        public GingerRunner Runner { get; set; }
+        private GingerRunner mRunner;
+        public GingerRunner Runner
+        {
+            get {
+                return mRunner;
+            }
+            set {
+                if(mRunner != value)
+                {
+                    mRunner = value;
+                    OnPropertyChanged(nameof(Context.Runner));
```

Comment:
no need to mention Context.xxx you already in the class, you can only to set nameof(Runner)

Reply:
Done